shared/api.go

```diff
@@ -41,6 +41,10 @@ func GetUsernameFromRequest(r *http.Request) string {
 	return subdomain
 }
 
+func ServeRobots() http.HandlerFunc {
```

Comment:
Is this being used anywhere?

Reply:
How did that get there? Was a commit from another branch that somehow leaked in. will remove.